docs/running-on-kubernetes.md

```diff
@@ -403,6 +403,36 @@ RBAC authorization and how to configure Kubernetes service accounts for pods, pl
 [Using RBAC Authorization](https://kubernetes.io/docs/admin/authorization/rbac/) and
 [Configure Service Accounts for Pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
 
+## Spark Application Management
+
+Like Mesos and standalone, Kubernetes provides simple application management via the spark-submit CLI tool in cluster mode.
+Users can kill a job by providing the submission ID that is printed when submitting their job.
+The submission ID follows the format ``namespace:driver-pod-name`` and is similar to the driverId found in standalone mode.
```

Comment:
Not sure why the reference to standalone here is helpful.

Reply:
Because k8s is a new backend and standalone prexisted for ages and people are familiar with it.